omegaconf/omegaconf.py

```diff
@@ -600,6 +604,34 @@ def to_container(
             resolve=resolve,
             enum_to_str=enum_to_str,
             exclude_structured_configs=exclude_structured_configs,
+            instantiate=instantiate,
+        )
+
+    @staticmethod
+    def to_object(
+        cfg: Any,
+        *,
+        resolve: bool = False,
```

Comment:
resolve should probably default to True here.

Reply:
Done in https://github.com/omry/omegaconf/pull/502/commits/872850b96ada2890a5ce45248e23c50eb23a72d5.